UnrealPlugin/TestingProject/Platforms/HoloLens/Config/HoloLensEngine.ini

```diff
@@ -0,0 +1,32 @@
+
+
+[/Script/HoloLensPlatformEditor.HoloLensTargetSettings]
```

Comment:
Looks like unreal just shoved this in. I'll see about stripping it out in the next PR. 👍

Reply:
Why not with a commit in this PR?

Reply:
Partially because we've been waiting on this test project for the pipeline for a while now, but moreso because a quick/simple PR here will be a good first proper run for the build gate when it's activated.